wagtail_localize/management/commands/import_po.py

```diff
@@ -0,0 +1,121 @@
+import pathlib
+from collections import defaultdict
+
+import polib
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+from wagtail.core.models import Site
+
+from wagtail_localize.models import Locale, default_locale_id
+from wagtail_localize.workflow.views.translate import MessageIngestor
+
+
+class LocaleLoader:
+    def __init__(self, locale, options):
+        self.locale = locale
+        self.options = options
+        self.items = {}
+        if self.exists:
+            infile = polib.pofile(self.filename, encoding='utf-8')
+            self.items = {
+                entry.msgid: entry.msgstr or entry.msgid for entry in infile
```

Comment:
`entry.msgstr or entry.msgid` is saving non translated text in the translated string in wagtail db,
otherwise required field constraints failed.

Furthermore, we may use the merge to avoid loosing translations but I am not
sur it is usefull in the workflow.

https://polib.readthedocs.io/en/latest/api.html#polib.POFile.merge